.env.example

```diff
@@ -145,6 +145,10 @@ LNPAY_API_KEY=LNPAY_API_KEY
 # Wallet Admin in Wallet Access Keys
 LNPAY_WALLET_KEY=LNPAY_ADMIN_KEY
 
+# AlbyWallet
+ALBY_API_ENDPOINT=https://api.getalby.com/
+ALBY_API_KEY=ALBY_ACCESS_TOKEN
```

Comment:
minor: I would still rename `ALBY_API_KEY` to `ALBY_ACCESS_TOKEN` because it is an OAuth access token.

Reply:
done